tests/unit/models/blueprint-test.js

```diff
@@ -23,6 +24,57 @@ var assign            = require('lodash/assign');
 var mkTmpDirIn        = require('../../../lib/utilities/mk-tmp-dir-in');
 
 var safeRestore = stub.safeRestore;
+var localsCalled;
+var normalizeEntityNameCalled;
+var fileMapTokensCalled;
+var filesPathCalled;
+var beforeUninstallCalled;
+var beforeInstallCalled;
+var afterInstallCalled;
+var afterUninstallCalled;
+
+var instrumented = {
+  locals: function(opts) {
+    localsCalled = true;
+    return this._super.locals.apply(this, arguments);
+  },
+
+  normalizeEntityName: function(name) {
+    normalizeEntityNameCalled = true;
+    return this._super.normalizeEntityName.apply(this, arguments);
+  },
+
+  fileMapTokens: function() {
+    fileMapTokensCalled = true;
+    return this._super.fileMapTokens.apply(this, arguments);
+  },
+
+  filesPath: function(opts) {
+    filesPathCalled = true;
+    return this._super.filesPath.apply(this, arguments);
+  },
+
+  beforeInstall: function(opts) {
+    debugger;
```

Comment:
@stefanpenner I don't think that `debugger` statement was supposed to stay there, right? :wink: